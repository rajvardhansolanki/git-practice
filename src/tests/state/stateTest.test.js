import { shallow } from "enzyme";
import React from "react";
import StateTest from "./stateTest";


// describe("state", () => {
//     it("check data", () => {
//         const wrapper = shallow(<StateTest />)
//         console.log(wrapper.debug())
//         expect(wrapper.find("ul").children().length).toEqual(34)
//     })
// })

describe("state", () => {
    it("snapshot", () => {
        const wrapper = shallow(<StateTest />)
        expect(wrapper).toMatchSnapshot()
    })
})