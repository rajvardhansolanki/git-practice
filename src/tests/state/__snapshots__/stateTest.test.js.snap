// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`state snapshot 1`] = `
<Fragment>
  <ul
    className="orderList"
  >
    <li
      className="abc"
      key="angular"
    >
      angular
    </li>
    <li
      className="abc"
      key="node"
    >
      node
    </li>
    <li
      className="abc"
      key="vew"
    >
      vew
    </li>
  </ul>
  <div
    className="sdsds"
    key="angular"
  >
    hello
  </div>
</Fragment>
`;
